packages/verification-vue-component/src/style/components/verification/card/_base.scss

```diff
@@ -78,6 +78,26 @@
                 + .section {
                     margin-top: 1rem;
                 }
+
+                &.retraction-note {
+                    display: flex;
+                    flex-direction: column;
+                    padding: 12px;
+                    border-radius: 8px;
+                    background-color: #e8f1fe;
+
+                    .note-label {
+                        padding-bottom: 5px;
+                        text-transform: uppercase;
+                        color: #0551bb;
+                        font-size: 13px;
+                        font-weight: 600;
+                    }
+
+                    .note-value {
+                        color: #434d64;
```

Comment:
Lots of new colory here, does it make sense to put them in variables? Why are there so many new colory anyway? It looks like another shade of blue.

Reply:
Normally I go through the CSS in the end and refactor it but just forgot to do it here. I introduced a new color "verifcation-pure-blue" (maybe you have a better name) because it's a different kind of blue we didn't have so far as I could see.

Reply:
Maybe @stephalba can help with a proper name. Is this an additional color, or should this blue replace the current blue in the future? If not, in which context is this blue used or should be used?

Reply:
We just discussed it, it's an additional color and in figma it's defined as "Info Message Text" and shouldn't replace an other color. I didn't use "$info-message-text" as name because the other blues have all a "general" blue name.

Reply:
Ok, makes sense. I would say it's an ok name for the moment. We need to make a concept for colors/color-names when we refactor anyway.